source/chapters/library.rst

```diff
@@ -894,6 +899,28 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _library-rekordbox:
+
+Using the Rekordbox library
+---------------------------
+To read Rekordbox databases for CDJ/XDJ players on USB or SD devices,
+the database has to be exported to a USB or SD device with a FAT or HFS
+file system using the Rekordbox Export mode. Instructions on how to do this can be found in this `article by Pioneer DJ <https://forums.pioneerdj.com/hc/en-us/articles/205021549-Moving-your-rekordbox-database-Video->`__ .
+Databases that have been moved to an external device via Preferences > Advanced > Database management are not supported.
+
+The following data is read:
+
+* Folders
+* Playlists
+* Beatgrids
+* Hot cues
+* Memory cues
+* Loops (not yet supported in Mixxx 2.3)
```

Comment:
I thought only the first loop was supported in 2.3?

Reply:
Yes, I just checked [the blog post about this](https://mixxx.org/news/2020-07-20-new-in-2-3-rekordbox-support/). Also, the manual should explain how Rekordbox's memory and hotcues are imported to Mixxx's hotcues.

Reply:
I just looked this up. You're right.  Since Mixxx currently has one loop, during importation, only the first chronological Rekordbox loop is mapped to the Mixxx loop. The rest of the loops are appended as Mixxx hotcues following the previously imported Rekordbox hotcues.

Reply:
> Yes, I just checked [the blog post about this](https://mixxx.org/news/2020-07-20-new-in-2-3-rekordbox-support/). Also, the manual should explain how Rekordbox's memory and hotcues are imported to Mixxx's hotcues.

Alright. Let me add this.

Reply:
I think the is paragraph should be merged with the "How Rekordbox X is Imported".